apps/dc_tools/odc/apps/dc_tools/s3_to_dc.py

```diff
@@ -29,16 +34,75 @@
     transform_stac,
     update_flag,
     update_if_exists_flag,
+    url_string_replace,
     verify_lineage,
-    publish_action,
 )
+from odc.aws import _aws_unsigned_check_env, auto_find_region, s3_client, s3_fetch
+from pystac import Item
 
 
 def doc_error(uri, doc) -> None:
     """Log the internal errors parsing docs"""
     logging.exception("Failed to parse doc at %s", uri)
 
 
+class SimpleFetcher:
```

Comment:
Should this perhaps live in `odc.aio` alongside `S3Fetcher`?

Reply:
Ok, I didn't realise it would have to be made async to be put there.

Reply:
Maybe it's just me being pedantic! But that's what the `aio` stands for :)

Reply:
It's not an asynchronous implementation (although perhaps it should be).